html/user/server_status.php

```diff
@@ -183,6 +185,11 @@ function show_status_html($x) {
         ";
     }
     end_table();
+
+    if ( isset($server_version) && isset($git_commit) ) {
+        echo "Server version: $server_version (<a href=https://github.com/BOINC/boinc/commit/$git_commit>$git_commit)</a>)<br>";
+    }
+
```

Comment:
The variables $server_version and $git_commit are not visible within the function.  I had to user

```
    global $server_version;
    global $git_commit;
    if ( isset($server_version) && isset($git_commit) ) {
        echo "Server version: $server_version (<a href=https://github.com/BOINC/boinc/commit/$git_commit>$git_commit)</a>)<br>";
    }
```
For it to render correctly.